rfc-165-add-api-endpoints-to-transition.md

```diff
@@ -25,16 +25,27 @@ ActiveRecord subclasses and dependency versions consistent.
 
 ## Proposal
 
-We should add API endpoints to Transition that will enable us to retrieve the
-data required for Bouncer to function.
+Bouncer should retrieve data from Transition via an API. This involves:
+- Adding API endpoints that allow us to retrieve the minimum amount of data for
+  Bouncer to function.
+- Removing database dependencies from Bouncer.
```

Comment:
Is it worth mentioning adding support for the Transition API to GDS API Adapters as a distinct part of the work of introducing the endpoints?